packages/app/src/components/search/SearchPage.tsx

```diff
@@ -55,12 +63,23 @@ const SearchPage = () => {
             </Paper>
           </Grid>
           <Grid item xs={3}>
+            <SearchType.Accordion
```

Comment:
I could go either way on whether or not we update `create-app` with this, since this is more of a choice for adopters.  It _is_ a more polished experience, though (IMO).

Reply:
I think its nice to update the app with these new things around search, as search is still a very new plugin, so let's do it! :)

Reply:
Makes sense!  Done here: https://github.com/backstage/backstage/pull/8646/commits/a7fdfa935a89f47e22afd15ccdeb9566146d2eed